src/components/board/Board.vue

```diff
@@ -145,6 +185,32 @@ export default {
 			this.loading = false
 		},
 
+		async ensureSession(boardId = this.id) {
+			if (this.token) {
+				try {
+					await sessionApi.syncSession(boardId, this.token)
+				} catch (err) {
+					// session probably expired, let's try again
+					// with a fresh session
+					this.token = null
+					setTimeout(() => {
+						this.ensureSession()
+					}, 100)
+				}
+			} else {
+				try {
+					const res = await sessionApi.createSession(boardId)
+					this.token = res.token
+				} catch (err) {
+					showError(err)
+				}
+			}
+		},
+
+		async refreshData() {
+			await this.$store.dispatch('refreshBoard', this.id)
```

Comment:
Might be a bit more lightweight to just fetch the sessions in the same request as it is renewed

Reply:
the periodic refreshing was there, to catch cases where some update events got lost like with suspended tabs.

with the recent updates we should catch almost everything, so I've removed it, so this should be obsolete :)